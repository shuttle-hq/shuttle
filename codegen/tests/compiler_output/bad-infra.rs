#[shuttle_codegen::main(something = "not right")]
async fn bad_infra() -> ShuttleRocket {}
