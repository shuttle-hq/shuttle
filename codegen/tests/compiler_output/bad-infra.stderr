error: Invalid macro attribute key: 'something'
 --> tests/compiler_output/bad-infra.rs:1:1
  |
1 | #[shuttle_codegen::main(something = "not right")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `shuttle_codegen::main` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compiler_output/bad-infra.rs:2:41
  |
2 | async fn bad_infra() -> ShuttleRocket {}
  |                                         ^ consider adding a `main` function to `$DIR/tests/compiler_output/bad-infra.rs`
